ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandTest.java

```diff
@@ -61,6 +62,7 @@ public void shouldSerializeDeserializeCorrectly() throws IOException {
     grep(serialized, ".*\"streamsProperties\" *: *\\{ *\"foo\" *: *\"bar\" *\\}.*");
     grep(serialized, ".*\"statement\" *: *\"test statement;\".*");
     grep(serialized, ".*\"originalProperties\" *: *\\{ *\"biz\" *: *\"baz\" *\\}.*");
+    grep(serialized, ".*\"version\" *: *\"" + Version.getVersion() + "\".*");
```

Comment:
Add test to ensure old json, without version, can be deserialized?

Reply:
That's what the first test does - I suppose what we need is to make sure that the new json can be deserialized properly 😂